renovate.json

```diff
@@ -5,7 +5,7 @@
     {
       "description": "Allow the locked in provider version to be updated to the latest for DAs",
       "enabled": true,
-      "matchFileNames": ["solutions/account-infrastructure-base/**"],
+      "matchFileNames": ["solutions/fully-configurable/**"],
```

Comment:
this rule is no longer needed, you can remove it. Its part of our common rules in `terraform-ibm-modules/common-dev-assets:commonRenovateConfig` now